ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/wizard/v1/wizard/clientlibs/contentframe/css/wizard.css

```diff
@@ -0,0 +1,72 @@
+<!--/*~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+  ~ Copyright 2024 Adobe
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~*/-->
+
+.cmp-adaptiveform-wizard{
+  
+}
+.cmp-adaptiveform-wizard__label{
+
+}
+.cmp-adaptiveform-wizard__label-container{
+  
+}
+.cmp-adaptiveform-wizard__tabList {
+}
+
+.cmp-adaptiveform-wizard__tab {
+    
+}
+
+.cmp-adaptiveform-wizard__tab--active {
+  
+}
+
+.cmp-adaptiveform-wizard__wizardpanel {
+    display: none;
+}
+
+.cmp-adaptiveform-wizard__wizardpanel--active {
+  display: block;
+}
+
+.cmp-adaptiveform-wizard__nextNav:hover{
+
+}
+.cmp-adaptiveform-wizard__nextNav{
+
+}
+
+.cmp-adaptiveform-wizard__previousNav:hover{
+
+}
+.cmp-adaptiveform-wizard__previousNav{
+
+}
+.cmp-adaptiveform-wizard__previousNav__hidden{
```

Comment:
Shouldn't hidden be a state modifier in BEM (--hidden) if not using data attribute. Btw, why not use the data attribute here?

Reply:
This BEM already exists, in this PR just enabling to load basic CSS ( like `display: none` of inactive tabs) in authoring.
Recently we did some refactoring to avoid loading runtime libraries in authoring, this was missed.

Reply:
@dmaurya929 Can we change this to modifier, this is incorrect